runtime/rasdump/javadump.cpp

```diff
@@ -1770,12 +1770,55 @@ JavaCoreDumpWriter::writeMemoryCountersSection(void)
 			writeNativeAllocator("Other", i + 1, 0, frame->liveBytes, frame->liveAllocations);
 		}
 	}
+        /* Write the section trailer */
+        _OutputStream.writeCharacters(
+                "NULL\n"
+                "NULL           ------------------------------------------------------------------------\n"
+        );
+
+#ifdef J9ZTPF
```

Comment:
Please use `#if defined(J9ZTPF)` and repeat the condition in the corresponding `#endif`:
```c
#if defined(J9ZTPF)
```
(etc.)
```c
#endif /* defined(J9ZTPF) */
```